app/services/case_information_service.rb

```diff
@@ -2,7 +2,7 @@
 
 class CaseInformationService
   def self.get_case_information(offender_ids)
-    CaseInformation.includes(:responsibility, :early_allocations, :local_delivery_unit).
-      where(nomis_offender_id: offender_ids).index_by(&:nomis_offender_id)
+    Offender.includes(case_information: [:responsibility, :early_allocations, :local_delivery_unit]).
+      where(nomis_offender_id: offender_ids).map(&:case_information).compact.index_by(&:nomis_offender_id)
   end
 end
```

Comment:
This entire service object is only used by one consumer – `Prison::OffenderEnumerator#enrich_offenders`.

We should probably just roll it into that. It doesn't make sense to have a whole service object just for this.